packages/amplify-cli-core/src/state-manager/stateManager.ts

```diff
@@ -206,6 +206,17 @@ export class StateManager {
     });
   };
 
+  private doesExist = (filePathGetter: any, projectPath?: string): boolean => {
```

Comment:
filePathGetter should be a `(projPath?: string) => string` rather than any

Reply:
ah, meant to change that. Good catch!